substrate-node/pallets/pallet-smart-contract/src/mock.rs

```diff
@@ -145,12 +145,12 @@ impl frame_system::Config for TestRuntime {
     type MaxConsumers = ConstU32<16>;
 }
 
-pub const EXISTENTIAL_DEPOSIT: u64 = 500;
+pub const EXISTENTIAL_DEPOSIT: u64 = 2;
```

Comment:
Why was this value changed?

Reply:
For testing another value different from runtime
But let s go back to 500 to have similar conditions